### DISCUSSION TITLE
I dont know how to use lazyvim :((

### DISCUSSION THREAD

**Discussion by n00dlephr0g** [olimorris/onedarkpro.nvim#247] [Q&A] [answered]
hi guys i just started moving into vim and would love to have this theme installed

I have kickstart installed but I have no idea what the block of code under the Lazy install section means

would someone be happy to walk me through where I'm cloning this repo, where to move folders and what to move and like where different lines of code goes?

**Comment by olimorris** [comment, marked as answer]
I'd recommend starting [here](https://www.youtube.com/results?search_query=lazy+vim)
